src/site/_includes/partials/analytics.njk

```diff
@@ -1,12 +0,0 @@
-{% set analyticsScript %}
-window.dataLayer = window.dataLayer || [];
-function gtag(){dataLayer.push(arguments);}
```

Comment:
We actually still need this file since some of the code references the global `gtag()` function (see #9277). Also, if/when we switch to GTM then we will need the global `dataLayer` variable, so I think it makes sense to keep everything but the `loadScript()` call here in this partial.

I'll add a commit with these changes.